logic/round.js

```diff
@@ -149,7 +149,11 @@ class Round {
 			const queries = votes.map(vote =>
 				self.t.rounds.updateVotes(
 					self.scope.modules.accounts.generateAddressByPublicKey(vote.delegate),
-					new Bignum(vote.amount).floor()
+					// Have to revert the logic to not use bignumber. it was causing change
+					// in vote amount. More details can be found on the issue.
+					// 		new Bignum(vote.amount).floor()
+					// TODO: https://github.com/LiskHQ/lisk/issues/2423
+					vote.amount
```

Comment:
Originally it was `Math.floor(vote.amount)` , so I think we should keep it that way
https://github.com/LiskHQ/lisk/blob/master/logic/round.js#L152

Reply:
Ok we can, but the amount can't be decimal so no need of floor here.

Reply:
Yep, is always nice to get rid of useless logic when we have an opportunity, and I already tested it without `Math.floor`.

Reply:
Done

Reply:
@4miners Yes I do agree with you. This time I didn't follow, just to get this PR merged quickly, so we can start testing further.

Reply:
mmm yea, it's unnecessary function call, but for reverting, i think it make sense to do just put it back, and when we are going to fix it we should fix it in the right way imo